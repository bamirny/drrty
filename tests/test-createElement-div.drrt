def int main(){
	createHTMLDocument("printbig.css");
	createElement("div","newel","This element will have a different color");
	return 0;
	
}