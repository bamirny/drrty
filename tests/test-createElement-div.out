<!DOCTYPE html><html><head><link rel="stylesheet" href="printbig.css"></head><body><div class="newel">This element will have a different color</div>
